### PR TITLE
Feature/35860 - a11y test

### PR DESCRIPTION
**Pull request by anqaka**
accessibility test for components added
library: [accessibilityjs](https://github.com/github/accessibilityjs)
to run: `gulp a11y`

Task: https://dev.snowdog.pro/issues/35860

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -79,6 +82,11 @@ gulp.task('fractal:build', ['inheritance', 'svg-sprite', 'sass'], () => {
 });
 
 // Gulp tasks
+gulp.task('a11y', () => {
+  fractal.components.set('default.preview', '@preview-test');
```
Comment (Igloczek) [inline gulpfile.js]: Let's rename it to `preview-accessibility` or `preview-a11y`, there is possibility that we will have more tests in the future

File: yarn.lock [hunk 2]
```diff
@@ -6,50 +6,7 @@
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/@allmarkedup/fang/-/fang-1.0.0.tgz#20aa2ea365c57b0a28e6b563613270f5630d6883"
 
-"@frctl/fractal@^1.0.0", "@frctl/fractal@^1.1.0-alpha.0":
```
Comment (Igloczek) [inline yarn.lock]: Weird... You didn't change any dependency but there are changes. Check if you have the latest version of Yarn and reinstall dependencies locally.

File: docs/styles/a11y-styles.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+@import 'docs-only-styles/checkbox';
+@import 'docs-only-styles/button';
+@import 'docs-only-styles/grid';
+@import 'docs-only-styles/icons';
+@import 'docs-only-styles/list';
+@import 'docs-only-styles/minicart-product';
+@import 'docs-only-styles/newsletter';
+@import 'docs-only-styles/product';
+@import 'docs-only-styles/range-filter';
+@import 'docs-only-styles/search';
```
Comment (Igloczek) [inline docs/styles/a11y-styles.scss]: It's kinda pointless to load docs-only-styles, b/c it doesn't work with a11y preview enabled.
The workaround will be to add `docs-only-styles` to the `a11y` preview too.

File: yarn.lock (hunk 2 above)
Comment (anqaka) [reply yarn.lock]: wierd, I have the latest version of yarn and I reinstall dependencies but nothing new in yarn.lock. When I've been working on this task I have tried some npm packages, maybe I commited it by mistake, should I fix it manually?

File: gulpfile.js (hunk 1 above)
Comment (anqaka) [reply gulpfile.js]: changed

**Pull request merged**